src/main/java/org/mbari/m3/dive/visualizer/AnnotationServiceHelper.java

```diff
@@ -0,0 +1,256 @@
+package org.mbari.m3.dive.visualizer;
+
+import io.helidon.webserver.Routing;
+import io.helidon.webserver.ServerRequest;
+import io.helidon.webserver.ServerResponse;
+import io.helidon.webserver.Service;
+
+import java.io.IOException;
+import java.net.URI;
+import java.net.http.HttpClient;
+import java.net.http.HttpHeaders;
+import java.net.http.HttpRequest;
+import java.net.http.HttpResponse;
+
+import java.util.Collection;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Set;
+
+import com.google.common.collect.Multiset.Entry;
+import com.google.gson.Gson;
+import com.google.gson.JsonArray;
+import com.google.gson.JsonObject;
+import com.google.gson.JsonParser;
+import com.google.gson.JsonElement;
+
+import org.mbari.expd.Dive;
+import org.mbari.expd.DiveDAO;
+import org.mbari.expd.jdbc.DiveDAOImpl;
+import org.json.simple.JSONObject;
+
+
+
+import io.helidon.webserver.Routing;
+
+public class AnnotationServiceHelper {
```

Comment:
I'm a little confused for the need of a separate AnnotationServiceHelper class. It looks to me like these methods can be moved into the primary AnnotationService class. Depending on what the methods were written for, each method can be an endpoint in our API. Any helper methods can be a private method. Come speak with me so we can figure out the next steps.